src/main/java/nl/esciencecenter/restape/ToolBenchmarkingAPIs.java

```diff
@@ -298,30 +397,50 @@ static boolean filterOutNonBioTools(List<String> openEBenchToolVersionURLs) {
    }
 
    /**
-    * Change the URL to retrieve detailed tool metrics rather than general tool
-    * information.
+    * Create a new list of URLs that references tool metrics rather than general
+    * tool
+    * information. In practice, the method replaces `/tool/` with `/metric/` in the
+    * URL.
     * 
     * @param openEBenchToolVersionURLs
-    * @return
+    * @return List of URLs that reference tool metrics.
     */
    static List<String> swapOEBCallTool2Metric(List<String> openEBenchToolVersionURLs) {
```

Comment:
Nit-picking again: I would have called it something with "replace", it's not really swapping.